content/docs/gke/gcp-e2e.md

```diff
@@ -60,7 +60,14 @@ is a 7.
 
 ### The overall workflow
 
-Here's an overview of what you accomplish by following this guide:
+The following diagram shows what you accomplish by following this guide:
```

Comment:
@sarahmaddox I think in the past we had discussed the overlap with the codelab;
https://codelabs.developers.google.com/codelabs/kubeflow-introduction/index.html?index=..%2F..index#0

Do you think it makes sense to continue to have the E2E guide on the website and the codelab or should we do one or the other?

Reply:
@jlewi I think we need to keep the E2E guide on the website. E2E guides are highly regarded (see the latest UX study which asks for more of them) and we don't want to send people all over the web to find them. The guide on the website is (or should be) curated and kept up to date, whereas the codelabs typically receive attention for major events/conferences but not between.